packages/assets/lib/getWidthHeight.js

```diff
@@ -8,8 +8,8 @@ module.exports = (resize) => {
     }
   }
   const [_width, _height] = resize.split('x')
-  const width = _width ? parseInt(_width) : null
-  const height = _height ? parseInt(_height) : null
+  const width = _width ? parseInt(Math.abs(_width)) : null
```

Comment:
`Math.abs` always return a number or NaN. No need to `parseInt` afterwards.

Generally `parseInt` without radix (2nd param) is [unsafe in browsers](https://coderwall.com/p/4eaixa/parseint-can-be-dangerous). While it's safe in node.js I'd still avoid it, just use the plus operator: `+'0023'`, it behaves as expected in all envs.

Reply:
Yes, but the puppeteer api explicitly expects an integer not a number on the width/height input. Without `parseInt` it could also be a float. As float values make no sense here anyway, I thought I add it globally.

Reply:
In that case: use `Math.floor` or `Math.ceil`.